internal/hcsoci/layers.go

```diff
@@ -120,10 +120,18 @@ func MountContainerLayers(ctx context.Context, layerFolders []string, guestRoot
 		if uvm.OS() == "windows" {
 			options := &hcsschema.VirtualSmbShareOptions{
 				ReadOnly:            true,
-				PseudoOplocks:       true,
 				TakeBackupPrivilege: true,
 				CacheIo:             true,
 				ShareRead:           true,
+				PseudoDirnotify:     true,
```

Comment:
NoLocks and PseudoDirNotify are new options we're adding for all layers?

Reply:
/ if you rebase the DefaultVSMBOptions function is available now also 😉

Reply:
fixed with new PRs.